mappings/net/minecraft/SharedConstants.mapping

```diff
@@ -109,9 +111,39 @@ CLASS net/minecraft/unmapped/C_otltijcb net/minecraft/SharedConstants
 		COMMENT else {@code (1 << {@link #SNAPSHOT_PROTOCOL_VERSION_EXTRA_BITS}) + {@link #SNAPSHOT_PROTOCOL_VERSION}}}
 	METHOD m_ltryxeto setGameVersion (Lnet/minecraft/unmapped/C_xyxorzsk;)V
 		ARG 0 gameVersion
+	METHOD m_mnjwgyqe stripInvalidCharacters (Ljava/lang/String;)Ljava/lang/String;
+		COMMENT {@return {@code text} with all {@linkplain #isValidChar invalid characters} stripped}
+		COMMENT <p>
+		COMMENT LF (linebreak; U+000A) is an invalid character and therefore stripped.
+		COMMENT Use {@linkplain #stripInvalidCharacters(String, boolean)} to keep linebreaks.
+		COMMENT
+		COMMENT @see #isValidChar(char)
+		COMMENT @see #stripInvalidCharacters(String, boolean)
+		ARG 0 text
 	METHOD m_pfbdxkbd isValidChar (C)Z
+		COMMENT {@return {@code true} if the character is not
+		COMMENT {@linkplain net.minecraft.util.Formatting#FORMATTING_CODE_PREFIX the formatting code prefix}
+		COMMENT (&bsol;u00a7),
+		COMMENT C0 control code (&bsol;u0000 to &bsol;u001f) or delete (&bsol;u007f),
+		COMMENT or {@code false} otherwise}
```

Comment:
Could this be rewritten to something like `{@return {@code false} if the character is an invalid one, or {@code true} otherwise}` and specify which characters are considered invalid in a different paragraph?

Reply:
Ok, modified and it should be more readable now!
![image](https://user-images.githubusercontent.com/12587332/184539838-f0c87a7b-e041-45e3-bd2b-c0f92a07e099.png)